### PR TITLE
LibPDF+pdf: Move Page to its own file; add --dump-contents flag to `pdf`

### PR DESCRIPTION
**Pull request by nico**

### CODE REVIEW

**Comment by nico** [comment]
Demo:

```
% time Build/lagom/bin/pdf --dump-contents ~/Downloads/sample.pdf
2 J
BT
0 0 0 rg
/F1 0027 Tf
57.3750 722.2800 Td
( A Simple PDF File ) Tj
ET
BT
/F1 0010 Tf
69.2500 688.6080 Td
( This is a small demonstration .pdf file - ) Tj
ET
BT
/F1 0010 Tf
69.2500 664.7040 Td
( just for use in the Virtual Mechanics tutorials. More text. And more ) Tj
...
```

File: Userland/Libraries/LibPDF/Page.cpp [hunk 1]
```diff
@@ -13,6 +13,10 @@ namespace PDF {
 PDFErrorOr<ByteBuffer> Page::page_contents(Document& document) const
 {
     // Table 3.27 Entries in a page object on Contents:
+    // "If this entry is absent, the page is empty. [...]"
+    if (contents.is_null())
```
Comment (trflynn89) [inline Userland/Libraries/LibPDF/Page.cpp:17]: Organizationally, I think it makes more sense for the commit to come before the `pdf` commit so it never has the chance to crash in the first place

**Review by trflynn89 — COMMENTED** [review]
just a couple nits

**Comment by nico** [comment]
Thanks!

**Review by trflynn89 — APPROVED** [review]

**Review by mattco98 — APPROVED** [review]

**Pull request merged**